prbt_hardware_support/src/modbus_adapter_brake_test.cpp

```diff
@@ -126,8 +129,10 @@ bool ModbusAdapterBrakeTest::sendBrakeTestResult(SendBrakeTestResult::Request& r
   }
 
   RegCont reg_cont(reg_block_size_, 0);
-  // Note: The FS controller needs a positive edge, so we first reset the
-  // registers by sending 0s.
+  // Note: The FS controller needs a positive edge, so we first reset the registers
+  reg_cont.at(reg_idx_cont_.at(modbus_api_spec::BRAKETEST_PERFORMED) - reg_start_idx_) = BRAKE_TEST_NOT_PERFORMED;
+  reg_cont.at(reg_idx_cont_.at(modbus_api_spec::BRAKETEST_RESULT) - reg_start_idx_) = BRAKE_TEST_NOT_PASSED;
```

Comment:
@jschleicher This is the breaking change, which also led to the `-1` reported in #379.